src/optsummary.jl

```diff
@@ -38,14 +38,36 @@ mutable struct OptSummary{T <: AbstractFloat}
     feval::Int
     optimizer::Symbol
     returnvalue::Symbol
-    nAGQ::Integer           # doesn't really belong here but I needed some place to store it
+    nAGQ::Integer           # don't really belong here but I needed a place to store them
```

Comment:
I would leave the singular verb and pronoun here.

Reply:
I meant to indicate that the two last values in the structure are not related to the optimization per se but are tucked away in here for lack of a better place to put them.